test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ApplicationInsights/ApplicationInsightsEndToEndTests.cs

```diff
@@ -103,26 +108,20 @@ private IHost ConfigureHost(LogCategoryFilter filter = null)
                 string expectedFunctionUserCategory = LogCategories.CreateFunctionUserCategory(testName);
```

Comment:
Won't AddApplicationInsights initialize this for us now?

Reply:
Perhaps I don't understand the question: we're trying to validate if the category is set on the telemetry. ApplicationInsights has no control over which categories are assigned. So, could you please elaborate?

Reply:
Hmm, it's sticking my comment on the wrong line somehow (I used VS tools to make my comments -- something must have been screwed up). I'll re-make the comment on the right line...

Reply:
My comment was meant to be around `SetActiveTelemetryConfiguration(host.Services.GetService<TelemetryConfiguration>());`, but it looks like you cleaned that up, which was exactly what I was asking for.

Reply:
yeah, Active lifetime is more than one test, so we have to clean it up after each run. I corrected it and moved to Dispose method